checkov/sca_package/scanner.py

```diff
@@ -86,32 +90,47 @@ async def run_scan_multi(
                     res['repository'] = str(input_paths_as_list[idx])
                     scan_results[idx] = res
 
-        return scan_results
+        # checking whether there are None results that indicates for error. id there is any, we return None
+        scan_results_without_nones: list[dict[str, Any]] = []
```

Comment:
Product question, in case we have an error in a single scan result, don't we still want to show all other results?

Reply:
good question. but as I understood, 
The logic here - if the results are partial, we have to terminate the run and exit code should be 2

@nimrodkor @gruebel am I right? or maybe we still want to report the result for the successful files?

Reply:
@itai1357 is correct